lib/ts/recipe/session/sessionFunctions.ts

```diff
@@ -84,13 +85,13 @@ export async function getSession(
                 recipeInstance,
                 accessToken,
                 handShakeInfo.jwtSigningPublicKey,
-                handShakeInfo.enableAntiCsrf && doAntiCsrfCheck
+                (await getEnableAntiCsrfBoolean(recipeInstance)) && doAntiCsrfCheck
```

Comment:
Can continue to use `handShakeInfo.enableAntiCsrf` based on our discussion

Reply:
resolved in commit [#308e21b ](https://github.com/supertokens/supertokens-node/pull/75/commits/308e21b0a70e438b12b999491df8cc66729d4bd1)